app/decorators/proxy_rule_decorator.rb

```diff
@@ -9,9 +9,8 @@ def pattern
     File.join('/', backend_api_path, pattern_value)
   end
 
-  def metric_system_name
-    object.metric.extended_system_name
-  end
+  delegate :metric, to: :object
```

Comment:
Hmmm if it is a decorator, why isn't it already delegated to the object?

Reply:
Belongs to https://github.com/3scale/porta/pull/1778